packages/components/src/components/sidebar-nav-collapsible/sidebar-nav-collapsible.css

```diff
@@ -14,7 +14,8 @@
   --border-bottom-color: var(--telekom-color-ui-additional-subtle);
   --border-left-color-third-nesting: var(--telekom-color-ui-additional-strong);
   --color-active: var(--telekom-color-text-and-icon-primary-pressed);
-  --color-primary: var(--telekom-color-text-and-icon-primary-standard);
+  --color-hover: var(--telekom-color-text-and-icon-primary-hovered);
```

Comment:
Hi Maomao, I updated the storybook documentation with the scoped variables so we should update those in the relevant story pls have a look here for reference https://3.basecamp.com/5297099/buckets/26183347/todos/4624032394